webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -192,6 +192,9 @@ export default {
     postMessage() {
       // Using a ref to the editor component and the getMessage method is mandatory to
       // be sure to get the most up to date value of the message
+      if (this.loading) {
```

Comment:
There is already the computed property postDisabled that should disable th ebutton once this.loading is true. It should be sufficient to prevent sending the message multiple times
maybe we need just to change the position of the instruction ```this.loading = true``` to be in first lines of this function ?  WDYT ?